yksgi/src/llvmbridge.rs

```diff
@@ -217,6 +237,12 @@ impl Value {
     pub fn kind(&self) -> LLVMValueKind {
         unsafe { LLVMGetValueKind(self.0) }
     }
+
+    pub fn get_name(&self) -> &str {
+        let mut size = MaybeUninit::<usize>::uninit();
+        let name = unsafe { CStr::from_ptr(LLVMGetValueName2(self.0, size.as_mut_ptr())) };
```

Comment:
Somewhere else in this code I just did:
```
        let mut size = 0;
        let name = CStr::from_ptr(LLVMGetValueName2(func, &mut size))
            .to_str()
            .unwrap();
```
Do we need to use `MaybeUninit` here? If so, we should change the other one too.

Reply:
Your suggestion works (just for now?), but the [Rust docs](https://doc.rust-lang.org/stable/std/mem/union.MaybeUninit.html#out-pointers) recommend using `MaybeInit` for out pointers, so I've gotten into the habit of using it.